### PR TITLE
move to web

### PR DESCRIPTION
**Pull request by soaptherope**

### CODE REVIEW

**Comment by soaptherope** [comment]
i really hope we dont need tests =)

File: src/main/java/org/andersen/Main.java [hunk 1]
```diff
@@ -1,87 +1,7 @@
...
-import org.andersen.service.impl.ReservationServiceImpl;
-import org.andersen.util.StateLoader;
-import org.andersen.util.StateSaver;
-
-import java.io.IOException;
-import java.util.List;
-import java.util.Optional;
-
 public class Main {
     public static void main(String[] args) {
```
Comment (Alexmansar) [inline src/main/java/org/andersen/Main.java]: do we still need it?

**Review by Alexmansar — CHANGES_REQUESTED** [review]
@soaptherope Hi)) in general good job)
about tests - we still can use mock for it ))

File: src/main/java/org/andersen/Main.java (hunk 1 above)
Comment (soaptherope) [reply src/main/java/org/andersen/Main.java]: why? i removed everything from main because i launch not main method but tomcat

**Comment by soaptherope** [comment]
added tests for the 2 controllers

File: src/main/java/org/andersen/Main.java (hunk 1 above)
Comment (soaptherope) [reply src/main/java/org/andersen/Main.java]: i create an apartment in web applications not in main

Comment (Alexmansar) [reply src/main/java/org/andersen/Main.java]: I mean, maybe we can remove whole class?

File: src/main/java/org/andersen/controller/ApartmentController.java [hunk 2]
```diff
@@ -1,71 +1,71 @@
...
-    private ApartmentService apartmentService;
-    private HotelService hotelService;
+        private ApartmentService apartmentService;
+        private HotelService hotelService;
 
-    @Override
-    public void init() {
-        ServletContext context = getServletContext();
+        @Override
+        public void init() {
```
Comment (Alexmansar) [inline src/main/java/org/andersen/controller/ApartmentController.java]: what again happened here?

File: src/test/java/controller/ApartmentControllerTest.java [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    @Mock
+    private HotelService hotelService;
+
+    @Mock
+    private ApartmentService apartmentService;
+
+    @BeforeEach
+    public void setUp() {
+        lenient().when(request.getServletContext()).thenReturn(servletContext);
+        lenient().when(request.getServletContext().getRequestDispatcher("/WEB-INF/jsp/apartments.jsp")).thenReturn(dispatcher);
```
Comment (Alexmansar) [inline src/test/java/controller/ApartmentControllerTest.java]: please, don't use lenient()
just use when where we  need it)

File: src/test/java/controller/ReservationControllerTest.java [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+    @Mock
+    private ServletContext servletContext;
+
+    @InjectMocks
+    private ReservationController reservationController;
+
+    @BeforeEach
+    public void setUp() {
+        lenient().when(request.getServletContext()).thenReturn(servletContext);
```
Comment (Alexmansar) [inline src/test/java/controller/ReservationControllerTest.java]: same

**Review by Alexmansar — CHANGES_REQUESTED** [review]
@soaptherope 
Hi)) good job)) Small comments))

**Review by Alexmansar — APPROVED** [review]
@soaptherope Hi) please, merge it))

**Pull request merged**